filebeat/filebeat.full.yml

```diff
@@ -395,22 +395,206 @@ output.elasticsearch:
   # Configure curve types for ECDHE based cipher suites
   #tls.curve_types: []
 
+#------------------------------- Kafka output ---------------------------------
+#output.kafka:
+  # The list of Kafka broker addresses to connect to.
+  #hosts: ["localhost:9092"]
+
+  # The Kafka topic used for produced events. If use_type is set to true, the
+  # topic will not be used.
+  #topic: beats
+
+  # Set Kafka topic by event type. If use_type is false, the topic option must
+  # be configured. The default is false.
+  #use_type: false
+
+  # The number of concurrent load-balanced Kafka output workers.
+  #worker: 1
+
+  # The number of times to retry publishing an event after a publishing failure.
+  # After the specified number of retries, the events are typically dropped.
+  # Some Beats, such as Filebeat, ignore the max_retries setting and retry until
```

Comment:
```
Some Beats, such as Filebeat, ignore the max_retries setting and retry until  all events are published. 
```
^^  Is this config file only for Filebeat? If so, can then this section can be deleted? This might remove some confusion when working between different beats.